oak_marketplace_emotion_recognition/README.md

```diff
@@ -0,0 +1,42 @@
+This contains the code for \***\*. For more information - visit [\*\***]()
+
+## Installation:
+
+1. Clone [repository](https://github.com/opencv-ai/oak-model-samples) with oak samples
```

Comment:
1. Clone the [repository](https://github.com/opencv-ai/oak-model-samples) with OAK model samples by the following command
    ```bash
    git clone https://github.com/opencv-ai/oak-model-samples.git
    ```